plugins/cache/cache_main.ml

```diff
@@ -298,16 +176,29 @@ let () =
          library.";
       `S "SEE ALSO";
       `P "$(b,regular)(3)";
-
     ] in
   let clean = Config.(flag "clean" ~doc:"Cleanup all caches") in
   let set_size = Config.(param (some int64) "size" ~docv:"N"
                            ~doc:"Set maximum total size of cached data to
                                  $(docv) MB. The option value will persist
                                  between different runs of the program") in
+  let set_overhead =
+    Config.(param (some float) "overhead"
+              ~doc:"Set overhead of maximum total size of cached data,
```

Comment:
Controls the aggressiveness of the garbage collector.  The higher the number the more space will be wasted but the cache system will run faster. It is expressed as a percentage of the max-size parameter.